eox_core/api/v1/views.py

```diff
@@ -116,7 +116,7 @@ class UserInfo(APIView):
     Auth-only view to check some basic info about the current user
     Can use Oauth2/Session/JWT authentication
```

Comment:
Update docstring, JWT authentication was removed

Reply:
Fixed in latest commit.